graylog2-web-interface/src/views/logic/slices/viewSlice.ts

```diff
@@ -102,12 +102,19 @@ const _recreateSearch = async (newView: View) => {
   return updatedView.toBuilder().search(updatedSearch).build();
 };
 
+export const executeSearch = () => (dispatch: AppDispatch, getState: () => RootState) => {
+  const view = selectView(getState());
+  const activeQuery = selectActiveQuery(getState());
+
+  return dispatch(execute(view.search, activeQuery));
+};
+
```

Comment:
I had to extract this logic from the `searchExecutionSlice`. I ended up with creating this function in the `viewSlice`, so we don't have to manually provide the search and active query where we called `execute` from the `searchExecutionSlice`.